src/components/ChooseMarkSwitch.tsx

```diff
@@ -0,0 +1,74 @@
+"use client";
+import React, { useContext, useState } from "react";
+import Image from "next/image";
+import cross from "@/icons/icon-x.svg";
+import nought from "@/icons/icon-o.svg";
+import { EMark, PlayersContext } from "@/context/playersContext";
+
+export default function ChooseMarkSwitch() {
+  const { players, setPlayers } = useContext(PlayersContext);
+  const [currentMark, setCurrentMark] = useState<EMark>(EMark.CROSS);
+
+  return (
+    <div className="bg-light-green rounded-xl p-5">
+      <div className="flex flex-col justify-between gap-4 items-center">
+        <div className="text-light-silver font-semibold">
+          PICK PLAYER 1&apos;S MARK
+        </div>
+        <div className="flex items-center bg-dark-green p-2 h-20 w-96 rounded-xl">
+          <button
+            onClick={() => {
+              setCurrentMark(EMark.CROSS);
+              const copyPlayerObj = { ...players };
+              copyPlayerObj.p1.mark = EMark.CROSS;
+              copyPlayerObj.p2.mark = EMark.NOUGHT;
+              setPlayers(copyPlayerObj);
+            }}
+            className={`flex-1 ${
+              currentMark === EMark.CROSS ? "p-3 bg-silver rounded-xl" : ""
+            }`}
+          >
+            <div className="flex justify-center">
+              <Image src={cross} alt="x" width="40" height="40" />
+            </div>
+          </button>
+          <button
+            onClick={() => {
+              setCurrentMark(EMark.NOUGHT);
+              const copyPlayerObj = { ...players };
+              copyPlayerObj.p1.mark = EMark.NOUGHT;
+              copyPlayerObj.p2.mark = EMark.CROSS;
+              setPlayers(copyPlayerObj);
+            }}
+            className={`flex-1 ${
+              currentMark === EMark.NOUGHT ? "p-3 bg-silver rounded-xl" : ""
```

Comment:
Should we install `clsx` or `classname`? to handle nicely the conditional class names

Reply:
classnames it is. But I don't understand why I should use the external library if the default condition statement also works.